p2p/src/net/types/mod.rs

```diff
@@ -162,8 +155,7 @@ pub enum SyncingEvent<T: NetworkingService> {
         /// Received request
         request: message::Request,
     },
-
-    /// Incoming response to a sent request
+    /// An incoming response to a sent request.
```

Comment:
Incredibly picky but the second half of the comment is redundant. It wouldn't be a response if it wasn't as a result of an outgoing request.